tests/foreman/virtwho/test_cli_virtwho.py

```diff
@@ -412,3 +412,49 @@ def test_positive_post_hypervisors(self):
                 )
             else:
                 self.assertTrue(result.status_code == 200)
+
+    @tier2
+    def test_positive_register_virt_who_host_with_uuid(self):
```

Comment:
@swadeley, actually, we have a case to cover  `hypervisor_id=uuid` already,  please refer to:
`def test_positive_hypervisor_id_option`

Reply:
OK, will check and test it.

Reply:
@swadeley any luck with that check? If you believe that test covers, would you mind closing this pr?

Reply:
No luck, dev unresponsive. Closing.